src/bin/tikv-server.rs

```diff
@@ -301,7 +306,8 @@ fn run_raft_server(pd_client: RpcClient, cfg: &TiKvConfig, security_mgr: Arc<Sec
     region_info_accessor.stop();
 
     if let Some(Err(e)) = worker.stop().map(|j| j.join()) {
-        info!("ignore failure when stopping resolver: {:?}", e);
+        info!("ignore failure when stopping resolver";
+              "error" => format!("{:?}",e));
```

Comment:
`"error" => ?e`

Reply:
It can cause an error which originates in a macro outside of the current crate.

Reply:
This is wired. @Hijiao Do you have ideas?

Reply:
This is wired. @Hijiao Do you have ideas?

Reply:
The error is 
```
error: cannot find macro `kv!` in this scope=======================> ] 321/322: tikv
   --> src/bin/tikv-server.rs:309:9
    |
309 | /         info!("ignore failure when stopping resolver";
310 | |               "error" => ?e);
    | |_____________________________^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: aborting due to previous error
```

Reply:
Any suggestion? @Hijiao

Reply:
Add `kv!` macro to `lib.rs` like [it](https://github.com/tikv/tikv/blob/bbb860f2265dafb6ed13df75d5f2a646db9154a2/src/lib.rs#L82), so you can simply use 
```info!("ignore failure when stopping resolver"; "err" => ?e); ``` 
Also, `"err"` is better than `"error"` here.